esrally/chart_generator.py

```diff
@@ -1453,25 +1453,39 @@ def generate_segment_memory(chart_type, race_configs, environment):
 def generate_dashboard(chart_type, environment, track, charts, flavor=None):
     panels = []
 
-    width = 6
-    height = 6
+    width = 24
+    height = 32
 
     row = 0
     col = 0
 
     for idx, chart in enumerate(charts):
+        panelIndex = idx + 1
+        # make index charts wider
+        if chart["_source"]["description"] == "index":
+            chart_width = 2 * width
+            # force one panel per row
+            next_col = 0
+        else:
+            chart_width = width
+            # two rows per panel
+            next_col = (col + 1) % 2
+
         panel = {
             "id": chart["_id"],
-            "panelIndex": idx,
-            "row": (row * height) + 1,
-            "col": (col * width) + 1,
-            "size_x": width,
-            "size_y": height,
-            "type": "visualization"
+            "panelIndex": panelIndex,
+            "gridData": {
+                "x": (col * chart_width),
+                "y": (row * height),
+                "w": chart_width,
+                "h": height,
+                "i": "{}".format(panelIndex)
+            },
+            "type": "visualization",
+            "version": "6.3.2"
```

Comment:
Adding a version was necessary to avoid that Kibana assumes a legacy format and attempts to load it in an outdated format. For newer Kibana versions we need to change the generator anyway as a couple of things have changed in 7.x.